packaging/linux/post_install.sh

```diff
@@ -37,6 +43,80 @@ make_mountpoint() {
   fi
 }
 
+systemd_exec_as() {
+    user=$1
+    shift
+
+    # Keybase supports running with and without systemd. In order for `su`
+    # to work with systemd commands, we need to pass in XDG_RUNTIME_DIR.
+    # /run/user/{uid} is the default when using systemd. We can't know the
+    # true value without installing systemd itself, which we don't want to
+    # depend on, so we use this default. Users who want to use a custom
+    # XDG_RUNTIME_DIR should specify it in their .profile so it is known
+    # by su -, which doesn't load pam_systemd. Unfortunately, this also
+    # results in an extra su invocation.
+    user_xdg_runtime_dir=""
+    # shellcheck disable=SC2016
```

Comment:
Remove?

Reply:
I like to leave these in so people in the future don't try to 'fix' something that's intentional

Reply:
Hmm, I have no idea what this line would do if its uncommented.  Can you add an explanation of what not to do?

Reply:
It's just a message to the shellcheck linter to ignore the fact that `'echo $XDG_RUNTIME_DIR'` doesn't expand since it's in single quotes on line 61. Without that line it would just show a lint warning if you have shellcheck enabled in your editor.